### PR TITLE
Problem: our app is not containerized

### LINKED ISSUE #18: Create  a Dockerfile

### PR DESCRIPTION
**Pull request by mehdisadeghi**
Solution: add Dockerfile Close #18

### CODE REVIEW

**Comment by carocad** [comment]
@mehdisadeghi I am not familiar with docker. So I can only ask: did you already tested this?

**Comment by mehdisadeghi** [comment]
@carocad yes, it worked just fine. It is nothing but an independent configuration file for Docker.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ FIXME: description
 
 Download from http://example.com/FIXME.
 
+### With Docker
```
Comment (carocad) [inline README.md]: :)

Comment (mehdisadeghi) [reply README.md]: We have to right something in the README eventually :)

Comment (carocad) [reply README.md]: definitely !!
that is on my todo list as well

**Pull request merged**